scriptworker/constants.py

```diff
@@ -179,7 +179,8 @@
 
     # docker-image cot
     "valid_docker_image_worker_types": (
-        "taskcluster-images",
+        "taskcluster-images",   # TODO: Remove this image once docker-images is the only valid worker type
+        "docker-images",
```

Comment:
I'm a little confused. iiuc, this should be gecko-images. I need to dive in more to understand.